wp-content/themes/core/components/blocks/icon_grid/src/Icon_Grid_Model.php

```diff
@@ -0,0 +1,18 @@
+<?php declare(strict_types=1);
+
+namespace Tribe\Project\Templates\Components\blocks\icon_grid\src;
+
+use Tribe\Libs\Field_Models\Models\Cta;
+use Tribe\Project\Blocks\Contracts\Block_Model;
+use Tribe\Project\Templates\Models\Collections\Icon_Collection;
+
+class Icon_Grid_Model extends Block_Model {
+
+	public string $title       = '';
```

Comment:
The should match the field names you defined in your Block_Config class, then they will get auto populated with the data.

Add a new field to the Block_Config, add it here and then it's automatically available in the controller's `$this->model` instance.